apps/web/cypress/tests/notification-editor/steps-actions.spec.ts

```diff
@@ -113,4 +113,70 @@ describe('Workflow Editor - Steps Actions', function () {
     cy.clickWorkflowNode(`node-inAppSelector`);
     cy.getByTestId(`step-should-stop-on-fail-switch`).should('be.checked');
   });
+
```

Comment:
it would be nice if you can add tests for the online filters ;)

Reply:
Done ;)
Added for all the unique filter types on UI